src/utils/config.ts

```diff
@@ -56,43 +47,14 @@ export const TEST = ENVIRONMENT === Environment.test;
 
 export const APP_VERSION = process.env.APP_VERSION;
 
-const DEFAULT_HOST = 'http://localhost:3000';
-
-export const BUILDER_HOST = {
-  name: Context.Builder,
-  url: new URL(process.env.BUILDER_CLIENT_HOST ?? DEFAULT_HOST),
-};
-
-export const PLAYER_HOST = {
-  name: Context.Player,
-  url: new URL(process.env.PLAYER_CLIENT_HOST ?? DEFAULT_HOST),
-};
-
-export const LIBRARY_HOST = {
-  name: Context.Library,
-  url: new URL(process.env.LIBRARY_CLIENT_HOST ?? DEFAULT_HOST),
-};
-
-export const ACCOUNT_HOST = {
-  name: Context.Account,
-  url: new URL(process.env.ACCOUNT_CLIENT_HOST ?? DEFAULT_HOST),
-};
+export const CLIENT_HOST = process.env.CLIENT_HOST ?? 'http://localhost:3000';
 
-export const ANALYTICS_HOST = {
-  name: Context.Analytics,
-  url: new URL(process.env.ANALYTICS_CLIENT_HOST ?? DEFAULT_HOST),
-};
+export const LIBRARY_HOST = process.env.LIBRARY_CLIENT_HOST ?? CLIENT_HOST;
 
-export const CLIENT_HOSTS = [BUILDER_HOST, PLAYER_HOST, LIBRARY_HOST, ACCOUNT_HOST, ANALYTICS_HOST];
+export const ALLOWED_ORIGINS = [new URL(CLIENT_HOST).origin, new URL(LIBRARY_HOST).origin];
```

Comment:
This can potentially fail if the CLIENT_HOST and LIBRARY_HOST are not correct URLs, should we catch and throw a better error ? I know by experience that such issues are hard to debug and find the root ...